modules/post-install-edge-aws-extend-cluster.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-tasks.adoc
+
+:_content-type: CONCEPT
+[id="post-install-edge-aws-extend-cluster_{context}"]
+= Extend existing clusters to use AWS Local Zones
+
+As a post-installation task, you can extend an existing {product-title} cluster on Amazon Web Services (AWS) to use AWS Local Zones.
+
+Extending nodes to Local Zone locations comprises the following steps:
+
+- Adjusting the cluster-network Maximum Transmission Unit (MTU)
```

Comment:
Seems like we're split on this across the repo (and even within this assembly), but it doesn't seem like "Maximum Transmission Unit" needs to be init-capped (a general networking term). Unless this was maybe referring to a specific UI label somewhere.

Reply:
Updated numerous entities.